routers/api/v1/api.go

```diff
@@ -820,6 +820,7 @@ func RegisterRoutes(m *macaron.Macaron) {
 									Post(reqToken(), bind(api.SubmitPullReviewOptions{}), repo.SubmitPullReview)
 								m.Combo("/comments").
 									Get(repo.GetPullReviewComments)
+								m.Post("/dismissals", bind(api.DismissPullReviewOptions{}), repo.DismissPullReview)
```

Comment:
Why not dismiss_review like the normal routes?

(We also need a request_review API it appears...)

Reply:
To follow github style. 
for request_review API, it's in #11355, Thanks :)